_submissions/round-12/0/2015-02-06-about-kai-yang.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2015-02-09
+round: Round 12
+title: About Kai Yang
+author: Kai Yang
+permalink: /2015/02/kai-yang/
+tags:
+  - Biography
+---
+
+# Kai Yang
```

Comment:
Please remove this line - the title of the post will be filled in automatically from the `title` field in the header when the page is compiled.